driver/conv_driver.hpp

```diff
@@ -896,7 +896,7 @@ std::vector<int> ConvDriver<Tgpu, Tref>::GetWeightTensorLengthsFromCmdLine()
         }
     }
 
-    miopenConvolutionMode_t mode;
+    miopenConvolutionMode_t mode = miopenConvolution;
```

Comment:
I do not like this way of initializing variable, however, compiler complains about it risking been uninitialized and used, probably due to Line 910 below is not properly parsed and understood by the compiler.

Reply:
If you want an explicitly invalid value you can do this:
```suggestion
    miopenConvolutionMode_t mode = (miopenConvolutionMode_t )-1;
```

Reply:
It will bring another error:
> error: use of old-style cast [-Werror,-Wold-style-cast]